### ISSUE TITLE
Some way of Tab History ?

### ISSUE THREAD

**Issue by konandrom** [goldendict/goldendict#871]
I am simultaneously working with two instances of GD, one as the dictionary tool we all use and love, the other, portable one, under Sandboxie, as a kind of dedicated Book reader. When opening this instance of GD, I have to manually restore the Tabs I am supposed to use. So my question is, whether there is a way to save the previously assigned Tabs from one session to another, as to be able to restore them the next time I fire up my PC. Some way of Tab or Favorites History... By the way, there are hundreds of files (=scanned book pages) I am working with.
![screenshot gd_1](https://user-images.githubusercontent.com/30632144/28874867-e31b94aa-7793-11e7-8b3d-4ab4bfd17058.jpg)

**Comment by KayLeung** [comment]
Yeah. Same here: https://github.com/goldendict/goldendict/issues/786
